src/fsharp/Interactive.DependencyManager/AssemblyResolveHandler.fs

```diff
@@ -50,20 +41,56 @@ type AssemblyResolveHandler (assemblyProbingPaths: AssemblyResolutionProbe) =
 
         with | _ -> Unchecked.defaultof<Assembly>
 
-#if NETSTANDARD
     let handler = Func<AssemblyLoadContext, AssemblyName, Assembly>(resolveAssemblyNetStandard)
     do AssemblyLoadContext.Default.add_Resolving(handler)
 
     interface IDisposable with
         member _x.Dispose() =
             AssemblyLoadContext.Default.remove_Resolving(handler)
 
-#else
+#endif
+
+/// Type that encapsulates AssemblyResolveHandler for managed packages
+type AssemblyResolveHandlerDeskTop (assemblyProbingPaths: AssemblyResolutionProbe) =
```

Comment:
```suggestion
type AssemblyResolveHandlerDesktop (assemblyProbingPaths: AssemblyResolutionProbe) =
```

Reply:
Desktop is like a proper word these days, it is no longer a portmanteau.  So I would prefer to reject this suggestion if that is Okay.

Reply:
So then what does `Top` mean in this circumstance?